### PR TITLE
Add custom CSS support

### PR DESCRIPTION
**Pull request by ga-hegsvold**
- Add _templates/css/sesam.css to contain custom CSS
- Update _templates/layout.html to use custom CSS
- Update README.md with custom CSS setup and usage instructions

### CODE REVIEW

**Review by gabriellsesam — APPROVED** [review]

File: training/README.md [hunk 1]
```diff
@@ -87,3 +87,44 @@ All topics in a chapter will be shown in the main Table of Contents, meaning tit
...
+Create a link to it from `_build/html/_static/css/`:
+
+```
+$ cd _build/html/_static/css
+$ ln -s ../../../../_templates/css/sesam.css sesam.css
+```
+
+This will ensure correct relative link ref.
+
+With the custom CSS in the expected folder, `_templates/layout.html` is now able include it when building
```
Comment (JonasAls89) [inline training/README.md:109]: typo: is now able to include it when ... Du mangler "to" i setningen

File: training/README.md [hunk 2]
```diff
@@ -87,3 +87,44 @@ All topics in a chapter will be shown in the main Table of Contents, meaning tit
 
 ## How does numbering work?
 Numbering is added in post as the docs page is generated. We use the toctree flag :numbered: in training.rst to enable this.
```
Comment (JonasAls89) [inline training/README.md:89]: toctree eller doctree ?

**Review by JonasAls89 — CHANGES_REQUESTED** [review]

File: training/README.md (hunk 1 above)
Comment (ga-hegsvold) [reply training/README.md]: fixed

File: training/README.md (hunk 2 above)
Comment (ga-hegsvold) [reply training/README.md:89]: Skrev ikke om dette nå, men "toctree" er nok riktig. Står for "table of contents tree".

Comment (JonasAls89) [reply training/README.md:89]: Takk for forklaringen.

**Review by JonasAls89 — APPROVED** [review]

**Pull request merged**